### PR TITLE
Makes knockdowns great again

### LINKED ISSUE #21126: You can crawl while under the effect of neurotoxin
Exactly what it says on the title. Thanks crawling

### LINKED ISSUE #21269: You can just crawl away from goliath tentacle knockdowns
#### Description of issue
- goliath makes friends with your footsies
- bad at DDR, fall over
- 0 threat because you can just lolcrawl away

[oversight]

#### Difference between expected and actual behavior
get stunned by goliath, watch in horror as it draws closer, chance of death
vs
get tripped by goliath, scoff and slide away

#### Length of time in which bug has been known to occur
#20994

#### Server revision
c17f20e557cc58456a7bc23ab552b5ce3b28d1dd

### PR DESCRIPTION
**Pull request by DamianX**
#20994 was a mistake
Stuns should be stuns, not stuns-but-you-can-kinda-avoid-being-punished-by-clicking-the-floor.
0% tested

fixes #21269
actually fixes #21126 (smh)

:cl:
 * tweak: Undid a change that let people crawl out of knockdowns. Tabling, window smashing, neurotoxin, and more now properly stun their victims. You can still crawl if you're knocked down by pain.

### CODE REVIEW

**Comment by ShiftyRail** [comment]
Eh, I liked that slips weren't a silly instant stun.

**Comment by ghost** [comment]
Crawling adds at least some counterplay to the otherwise binary combat.

**Comment by DamianX** [comment]
Making "uncommon" stuns such as slipping or stepping your naked feet on a thorny plant counterable by crawling just forces people into the boring stuns meta (e.g. carry a taser/stun baton or get fucked). Change my mind.

**Comment by bathosbathtime** [comment]
I like knockout being different than stun, but it is clear most if not all of there were made when they were functionally the same.  The electropack not stunning, for example, is odd.
The stun value could also be less than the knockdown so that the hard stun is still there but the target gets some "soft" counter-ability for the more unique cases.

**Comment by Intigracy** [comment]
~~Bad change.~~

~~If you want things to stun, make them stun.~~

nvm I'm retarded and thought you made knockdown make you unable to crawl before reading the changes, this is fine

File: code/game/mecha/equipment/weapons/weapons.dm [hunk 1]
```diff
@@ -137,6 +137,7 @@
 		M.stuttering += 20
 		M.ear_deaf += 30
 		M.Knockdown(3)
+		M.Stun(3)
```
Comment (Intigracy) [inline code/game/mecha/equipment/weapons/weapons.dm]: Remove this one, it already has a chance to stun them just below it and is notoriously bad for stun spamming people

**Review by Intigracy — DISMISSED** [review]

**Comment by Intigracy** [comment]
honestly stuns should be shorter than knockdowns so you have an initial period where you can't do anything then you can crawl before getting back up

**Comment by DamianX** [comment]
I have no opinion regarding that last point, but I'll point out that the opposite is in effect currently, with certain slips stunning for more than than the knockdown duration.

**Comment by Sooose** [comment]
At least this will finally fix people being able to crawl out of a choke.

@ShiftyRail You can't crawl out of a slip at the moment regardless.

**Pull request merged**